tests/MongoDB.Driver.Tests/JsonDrivenTests/JsonDrivenConfigureFailPointTest.cs

```diff
@@ -48,13 +46,13 @@ protected override async Task CallMethodAsync(CancellationToken cancellationToke
 
         protected virtual IServer GetServer()
         {
-            var cluster = _client.Cluster;
+            var cluster = TestRunner.FailPointClient.Cluster;
```

Comment:
This method should probably be renamed to `GetFailPointServer`.

Reply:
This way of using of properties really confuses me... I keep looking for a static class called TestRunner and it doesn't exist...